config.py

```diff
@@ -1,2 +1,2 @@
-FULLNODE = "http://node1.puyuma.org:14266"
+FULLNODE = "http://node01.testnet.iotatoken.nl:16265"
```

Comment:
Why would you change this?

Reply:
A syncing issue in our testnet node because some neighbors is down, it's a workaround will change back soon.

Reply:
Try to minimize the necessary changes. Explain the crucial parts always.